### ISSUE TITLE
Remembered Azure Artifacts PAT is stored in Windows Credential Manager in plain text

### ISSUE THREAD

**Issue by thegrahamking** [NuGetPackageExplorer/NuGetPackageExplorer#1468]
**Type** (choose one):

- Bug

**NPE version**: 6.0.27+d98979f0a5

**OS version**: Windows 10 - 1904

**Installed from**: Microsoft Store

In case of a BUG:

- What is the current result?
PAT is stored in username field in Windows Credential Manager, i.e. in plain text
- What is the expected result?
PAT is stored in password field in Windows Credential Manager, i.e. encrypted
- Are there any workarounds?
No
- Is there a version in which it did work?
Unknown

**Comment by clairernovotny** [comment]
Hi, not sure what's meant by this -- text in the Windows Cred Manager is stored encrypted by the Cred API. When you view it in the credential manager, it decrypts and displays it. This is all by design.

**Comment by thegrahamking** [comment]
That is all true, however, NuGet Package Explorer asks for the PAT via the username field which _isn't_ encrypted because it's used as the _name_ of the credential rather than the _value_.
